src/pages/lib/graphqlapi/connect/q/platform/[platform].mdx

```diff
@@ -0,0 +1,8 @@
+export const meta = {
```

Comment:
Added so can more quickly review via localhost - please note that this is NOT the final structure of the page - only for reviewing purposed atm.